### PR TITLE
feat(PKD2007): add new standard and NACE2 concordance

### LINKED ISSUE #79: feat: Add POL PKD codes
## Description
Add POL PKD (business purpose/operation) codes.

### PR DESCRIPTION
**Pull request by maddyobrienjones**
## Description
adding PKD2007 standard and PKD2007-NACE2 concordance
closes #79 

## Types of changes

What types of changes does your code introduce:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: pyisic/_standards/pkd2007/pkd2007.py [hunk 1]
```diff
@@ -0,0 +1,1661 @@
+# -*- coding: utf-8 -*-
+"""`PKD2007 Standard <https://stat.gov.pl/en/metainformation/classifications/#Polish%20Classification%20of%20Activities%20(PKD)>`_.
+"""
+from ...types import Category, Classification, Standard, Standards
+
+PKD2007 = Standard(
```
Comment (JSPromisel) [inline pyisic/_standards/pkd2007/pkd2007.py:6]: Looks like @csmith-sayari found this in [English](https://github.com/sayari-analytics/sayari-graph/blob/b3e9c5017e4ef5c530ae23fb3371a52e8fd2903a/graph_parsers/POL/biznes/website_dictionary.py#L7), can we see if we can find that mapping?

Comment (JSPromisel) [reply pyisic/_standards/pkd2007/pkd2007.py:6]: Seeing English [here](https://stat.gov.pl/files/gfx/portalinformacyjny/en/defaultstronaopisowa/3249/1/1/pkd2007_en.pdf)

**Review by JSPromisel — CHANGES_REQUESTED** [review]

File: pyisic/_standards/pkd2007/pkd2007.py (hunk 1 above)
Comment (maddyobrienjones) [reply pyisic/_standards/pkd2007/pkd2007.py:6]: yep, i will switch to that

**Review by JSPromisel — APPROVED** [review]

**Comment by JSPromisel** [comment]
@maddyobrienjones did the checks run for previous commits?

**Comment by maddyobrienjones** [comment]
> did the checks run for previous commits?

doesn't seem like it, just the codeql one

**Comment by JSPromisel** [comment]
I checked the workflow files, GitHub status page and the branch protection settings and did not see any reason they should not run. The CodeQL step is new to me but I think it's standard for public repos. @nyoungstudios any thoughts on why the tests are hung up?

**Review by JSPromisel — APPROVED** [review]

**Pull request merged**